lib/polyfill/media_capabilities.js

```diff
@@ -24,6 +24,7 @@ shaka.polyfill.MediaCapabilities = class {
    * @export
    */
   static install() {
+    shaka.polyfill.MediaCapabilities.memoizedMediaKeySystemAccessRequests_ = {};
```

Comment:
Please move this declaration outside the install() function, and give it a type annotation.

Reply:
:+1: